avm/res/insights/scheduled-query-rule/tests/e2e/max/dependencies.bicep

```diff
@@ -7,18 +7,51 @@ param managedIdentityName string
 @description('Required. The name of the Log Analytics Workspace to create.')
 param logAnalyticsWorkspaceName string
 
+@description('Required. The name of the Action Group to create.')
+param actionGroupName string
+
 resource managedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2018-11-30' = {
   name: managedIdentityName
   location: location
 }
 
-resource logAnalyticsWorkspace 'Microsoft.OperationalInsights/workspaces@2021-06-01' = {
-  name: logAnalyticsWorkspaceName
+module logAnalyticsWorkspace 'br/public:avm/res/operational-insights/workspace:0.11.1' = {
```

Comment:
any reason for using the module here?

Reply:
Jap, see the role assignment 💪

Reply:
Didn't we have a spec/design decision for not using modules? Or is it just up to the author?

Reply:
It's up to the author. We discussed this a long time back and ended up agreeing: If it's a versioned module, it's ok. Have maybe seen half a dozen cases since then.